lab-cameron/routes/move-routes.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const createError = require('http-errors');
+const moveCtrl = require('../controllers/move-controller');
+
+module.exports = function(router) {
+  router.get('/move/:id', (req, res) => {
+    if (!req.params.id) return res.status(400).send(createError(400, 'bad request'));
+    moveCtrl.fetchMove(req.params.id)
+    .then(move => {
+      res.json(move);
```

Comment:
When doing mult-line ``then/catch`` blocks, you need to put the word ``return`` explicitly or else whatever data you're trying to return won't be passed through. Lines 10-11 should be:

```
.then(move => {
  return res.json(move);
}
```

In the alternative, you could make this a one-liner to avoid having to put the word "return."

```
.then(move => res.json(move));
```